components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -584,6 +585,16 @@ def validate_webui_config(
     validate_port(f"{WEBUI_COMPONENT_NAME}.port", clp_config.webui.host, clp_config.webui.port)
 
 
+def validate_mcp_server_config(clp_config: CLPConfig, logs_dir: pathlib.Path):
+    if not logs_dir.exists():
+        error_msg = f"{MCP_SERVER_COMPONENT_NAME} logs directory at {logs_dir!s} is missing."
+        raise ValueError(error_msg)
+
```

Comment:
instead of checking whether `logs_dir` is already a directory, can we check if the path could be a directory (so later we can mkdir at the location) like other validators? maybe we can simply use the helper `_validate_log_directory`?